packages/node-feature-discovery/v0.16.2+1/package.yaml

```diff
@@ -0,0 +1,16 @@
+# yaml-language-server: $schema=https://glasskube.dev/schemas/v1/package-manifest.json
+
+name: "node-feature-discovery"
+shortDescription: "Detects hardware features available on each node in a Kubernetes cluster,
+  and advertises those features using node labels."
+iconUrl: "https://avatars.githubusercontent.com/u/1728152"
+defaultNamespace: "node-feature-discovery"
+helm:
+  repositoryUrl: "https://kubernetes-sigs.github.io/node-feature-discovery/charts"
+  chartName: "node-feature-discovery"
+  chartVersion: "0.16.2"
+references:
+  - label: "ArtifactHub"
+    url: "https://artifacthub.io/packages/helm/node-feature-discovery/node-feature-discovery"
+  - label: "GitHub"
+    url: "https://github.com/NVIDIA/gpu-operator/tree/main/deployments/gpu-operator/charts/node-feature-discovery"
```

Comment:
This reference links to the wrong helm chart. 
This is the actual source: https://github.com/kubernetes-sigs/node-feature-discovery
The helm chart sources are here: https://github.com/kubernetes-sigs/node-feature-discovery/tree/master/deployment/helm/node-feature-discovery